src/backend/distributed/commands/multi_copy.c

```diff
@@ -631,7 +651,10 @@ CopyToNewShards(CopyStmt *copyStatement, char *completionTag, Oid relationId)
 										copyStatement->options);
 
 	CopyOutState copyOutState = (CopyOutState) palloc0(sizeof(CopyOutStateData));
-	copyOutState->binary = true;
+	copyOutState->delim = delimiterCharacter;
+	copyOutState->null_print = nullPrintCharacter;
+	copyOutState->null_print_client = nullPrintCharacter;
```

Comment:
I added it for that reason, hide regular output and only see errors and warnings and my setup was working for quite bit of time. I am not sure why it stopped to show warnings. Maybe some other stuff affected it, but I dont know at the moment.